fda-model/alternativeComparisonReport/AlternativeComparisonReport.cs

```diff
@@ -123,7 +133,9 @@ private static ConsequenceDistributionResult IterateOnConsequenceDistributionRes
                         if (_completedIterations % progressChunks == 0)//need an atomic integer count here.
                         {
                             double percentcomplete = _completedIterations / (double)_ExpectedIterations * 100;
-                            damageReducedResult.ReportProgress(damageReducedResult, new ProgressReportEventArgs((int)percentcomplete));
+                           //TODO we need to remove the hacked in progress reporting 
```

Comment:
And is this still a valid todo? The one above said the hack was fixed.

Reply:
This is a valid to do - this method carries out iteration for just one consequence distribution result, of which there may be many

Reply:
so we need to figure out how to handle the other dimensions in progress reporting